listbox.go

```diff
@@ -1,8 +1,8 @@
 package tinygl
 
 import (
-	"github.com/aykevl/tinygl/font"
 	"tinygo.org/x/drivers/pixel"
+	font "tinygo.org/x/tinygl-font"
```

Comment:
This works just as well (the package name is `font`):

```suggestion
	"tinygo.org/x/tinygl-font"
```

But the existing code is fine too if you prefer it for clarity.

Reply:
Seems like `go fmt` wants to change it to be that way? My editor keeps changing it to be how it is in the PR.

Reply:
I suppose it does that for clarity/readability.